src/contrib/cluster/Akka.Cluster.Sharding/ShardRegion.cs

```diff
@@ -369,55 +372,54 @@ private StopTimeoutWarning()
         /// <param name="typeName">TBD</param>
         /// <param name="settings">TBD</param>
         /// <param name="coordinatorPath">TBD</param>
-        /// <param name="extractEntityId">TBD</param>
-        /// <param name="extractShardId">TBD</param>
+        /// <param name="messageExtractor"></param>
         /// <returns>TBD</returns>
         internal static Props ProxyProps(
             string typeName,
             ClusterShardingSettings settings,
             string coordinatorPath,
-            ExtractEntityId extractEntityId,
```

Comment:
Removing old delegates and replacing with `IMessageExtractor` instead, for `ShardRegionProxy` and `ShardRegion` implementations.